openstack/objectstorage/v1/containers/requests.go

```diff
@@ -123,15 +124,18 @@ func Create(c *gophercloud.ServiceClient, containerName string, opts CreateOptsB
 
 // BulkDelete is a function that bulk deletes containers.
 func BulkDelete(c *gophercloud.ServiceClient, containers []string) (r BulkDeleteResult) {
-	// urlencode container names to be on the safe side
-	// https://github.com/openstack/swift/blob/stable/train/swift/common/middleware/bulk.py#L160
-	// https://github.com/openstack/swift/blob/stable/train/swift/common/swob.py#L302
-	encodedContainers := make([]string, len(containers))
-	for i, v := range containers {
-		encodedContainers[i] = v
```

Comment:
The comment says we encoded, but we didn't! The strings were just copied into a new slice :shrug: